alfresco-transform-core-aio/alfresco-transform-core-aio-boot/Dockerfile

```diff
@@ -5,7 +5,7 @@
 # ImageMagick is from ImageMagick Studio LLC. See the license at http://www.imagemagick.org/script/license.php or in /ImageMagick-license.txt.
 # alfresco-pdf-renderer uses the PDFium library from Google Inc. See the license at https://pdfium.googlesource.com/pdfium/+/master/LICENSE or in /pdfium.txt.
 
-FROM alfresco/alfresco-base-java:jre11-centos7-202205121725
```

Comment:
We need to use the same base images as the ones from this commit: https://github.com/Alfresco/alfresco-transform-core/commit/849ce8df3b9af600c28cbc476d7938e667f4f523#diff-852ca35bf59bc0b1dc4561fcd1fb174c41e50c7be83108073df949efc1ef2a42R8. Simply switching from JRE11 to JRE17, but keeping the distro (`Rocky Linux 8`) and the tagged date (`202207110835`) untouched.

Team 5 is working on migrating from Centos 7 to Rocky Linux 8, so going back to Centos 7 here would mean that we're reverting their work, which is something that we don't want to do.

Moreover, the tagged date is necessary to have reproducible Docker image builds, whereas the image tag without date is a moving target that will keep referencing the latest image that's been released for that combination of JRE + distribution. It's not ideal to use for content that we need to be able to build and distribute in a reproducible manner.

Reply:
Thank you for pointing that!

Reply:
Done :D